pkg/strategy/xgap/strategy.go

```diff
@@ -446,3 +446,13 @@ func quantityJitter2(q, maxJitterQ fixedpoint.Value) fixedpoint.Value {
 	randQuantity := fixedpoint.NewFromFloat(q.Float64() + rg*rand.Float64())
 	return randQuantity
 }
+
+func adjustPrice(price fixedpoint.Value, pricePrecision int) fixedpoint.Value {
+	if pricePrecision <= 0 {
+		return price
+	}
+	rate := math.Pow(10, float64(-pricePrecision))*0.1 + 1
+	priceAdjusted := util.RoundAndTruncatePrice(price.Mul(fixedpoint.NewFromFloat(rate)), pricePrecision)
+	log.Infof("adjusted price: %s -> %s", price.String(), priceAdjusted.String())
```

Comment:
log the price from the caller, not inside the function?

Reply:
Done.